paper/main.bib

```diff
@@ -1,12 +1,26 @@
-
-
 @misc{BickEtal07,
 	author = {I Bicking and B G{\'{a}}bor and {Python Packaging Authority}},
 	howpublished = {\url{https://github.com/pypa/virtualenv}},
 	month = {September},
 	title = {{v}irtualenv: Virtual {P}ython {E}nvironment builder},
 	year = {2007}}
 
+@misc{Chatify,
+	author = {{Chatify~Team}},
```

Comment:
On Zenodo, the authors field appears as "Jeremy Manning, Hemu, & Konrad Kording". 

So here I think it should be `J R Manning and H Manjunatha and K P K{\"{o}}rding`